clients/java/signalr/src/main/java/IHubProtocol.java

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+public interface IHubProtocol {
```

Comment:
We need to model this after the JS client APIs since we're not going to be handling partial messages.

Reply:
Also, Java naming style is to **not** use a prefix for interface names. So this should just be called `HubProtocol`

Reply:
Oh yeah. No prefix

Reply:
I think I prefer that over the .NET style, which smells too much like ye-olde-COM. But there's no changing .NET style now 😄